rust/gui-client/src-tauri/src/client/deep_link/linux.rs

```diff
@@ -25,17 +26,6 @@ impl Server {
 
         let listener = UnixListener::bind(&path).context("Couldn't bind listener Unix socket")?;
 
-        // Figure out who we were before `sudo`, if using sudo
-        if let Ok(username) = std::env::var("SUDO_USER") {
-            // chown so that when the non-privileged browser launches us,
-            // we can send a message to our privileged main process
-            std::process::Command::new("chown")
-                .arg(username)
-                .arg(&path)
-                .status()
-                .context("couldn't chown Unix domain socket")?;
-        }
-
```

Comment:
This is all gone because we aren't using sudo on the GUI after the process separation change.